docs/index.md

```diff
@@ -190,19 +214,56 @@ Default is `None`.
 
 ### JWT_PRIVATE_KEY
 
-This is an object of type `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey`. It will be used to sign the signature component of the JWT. It will override `JWT_SECRET_KEY` when set. Read the [documentation](https://cryptography.io/en/latest/hazmat/primitives/asymmetric/rsa/#cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey) for more details. Please note that `JWT_ALGORITHM` must be set to one of `RS256`, `RS384`, or `RS512`.
+Can be a scalar or a dict.
+
+When a dict, the dict key is taken as the JWT key id and the values as
+the key, e.g.:
+
+```python
+{ "kid": key }
+```
+
+The scalar or the dict value must be in any [private key format supported by PyJWT](https://github.com/jpadilla/pyjwt/blob/master/docs/algorithms.rst), for example of the types
+
+* `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey`
+* `cryptography.hazmat.primitives.asymmetric.ec.EllipticCurvePrivateKey`
+
+And will be used to sign the signature component of the JWT if `JWT_ALGORITHM` is set to any of the [supported algorithms](https://github.com/jpadilla/pyjwt/blob/master/docs/algorithms.rst)  other than the hash types `HS*`.
 
 Default is `None`.
 
 ### JWT_PUBLIC_KEY
 
-This is an object of type `cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKey`. It will be used to verify the signature of the incoming JWT. It will override `JWT_SECRET_KEY` when set. Read the [documentation](https://cryptography.io/en/latest/hazmat/primitives/asymmetric/rsa/#cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKey) for more details. Please note that `JWT_ALGORITHM` must be set to one of `RS256`, `RS384`, or `RS512`.
+Can be a scalar, a list or a dict.
+
+* When a dict, the dict keys are taken as the JWT key ids and the values as
+  keys, e.g.:
+
+  ```python
+  { "kid1": key1, "kid2": key2, ... }
```

Comment:
If these settings only contain `kid`s, why use a dictionary instead of an iterable (tuple, list)?

Reply:
For efficient lookup by key id